conda/recipes/rapids-logger/recipe.yaml

```
@@ -18,6 +18,11 @@ build:
     # Run tests from the build directory since they're cheap.
     ctest --test-dir build/ --output-on-failure
     cmake --install build/
+    # Check that no undefined symbols are present in the shared library
+    LIB="build/librapids_logger.so"
+    test -f build/librapids_logger.so
+    ./ci/check_symbols.sh build/librapids_logger.so
```

```suggestion
    LIB="build/librapids_logger.so"
    test -f build/${LIB}
    ./ci/check_symbols.sh build/${LIB}
```

Did you mean to re-use this variable like this? Otherwise, it wouldn't have any effect on that script (because it's not being `export`'d).

---

I did indeed, thanks for the catch.